File: app/src/main/java/com/fthdgn/books/api/response/Copy.java
package com.fthdgn.books.api.response;

import android.support.annotation.Keep;

import java.util.Date;

@Keep
@SuppressWarnings({"unused", "WeakerAccess"})
public class Copy {

    public Integer remainingCharacterCount;
    public Integer allowedCharacterCount;
    public String limitType;
    public Date updated;

}

File: app/src/main/java/com/fthdgn/books/api/response/Copy.kt
@file:Suppress("unused")

package com.fthdgn.books.api.response

import android.support.annotation.Keep
import java.util.*

@Keep
class Copy {

    var remainingCharacterCount: Int? = null
    var allowedCharacterCount: Int? = null
    var limitType: String? = null
    var updated: Date? = null

}
